subsys/net/lib/aws_fota/src/aws_fota.c

```diff
@@ -241,13 +256,14 @@ static int job_update_accepted(struct mqtt_client *const client,
 		execution_state = AWS_JOBS_SUCCEEDED;
 		err = update_job_execution(client, job_id,
 				execution_state, fota_state,
+				stored_offset,
 				execution_version_number, "");
 		if (err) {
 			LOG_ERR("Unable to update the job execution");
 			return err;
 		}
-	} else if (execution_state == AWS_JOBS_SUCCEEDED &&
-			fota_state == APPLY_UPDATE) {
+	} else if (execution_state == AWS_JOBS_SUCCEEDED
```

Comment:
Would it make sense to have some handling for the case when none of these apply?

Reply:
To be honest I don't think so since the default is to increment the execution_version_number and return.